backend/config/index.js

```diff
@@ -1,3 +1,11 @@
-const config = require('./variables.json');
+// const config = require('./variables.json');
```

Comment:
Hey @AmeyaJain-25 as we are using env variables now so let's remove this import. Also let's remove the `variables.json` file as we won't need it anymore

Reply:
Yes Sure.
As I mentioned, you have to delete it from your side. If I delete, then as it's already committed, it will come in the push right?